### ISSUE TITLE
Commands don't work on Window (Error: Cannot find module '...\npm\node_modules\gatsby\bin\cli-new' on `gatsby new`)

### ISSUE THREAD

**Issue by awesomeaniruddh** [gatsbyjs/gatsby#80]
`Gatsby` installed successfully (no `node-gyp` errors), but creating a new site is throwing the error at me. I'm using Windows 10. Reinstalling `gatsby` didn't fix it.

Full error:
```
C:\Users\aniru>gatsby new test
module.js:339
    throw err;
    ^

Error: Cannot find module 'C:\Users\aniru\AppData\Roaming\npm\node_modules\gatsby\bin\cli-new'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:134:18)
    at node.js:961:3
```

**Comment by KyleAMathews** [comment]
I'm not at all familiar with NPM on Windows... perhaps this StackOverflow answer will help? Please report back if you figure it out! http://stackoverflow.com/questions/9587665/nodejs-cannot-find-installed-module-on-windows

**Comment by MichaelDemchenko** [comment]
on windows 10
need rename files
C:\Users\aniru\AppData\Roaming\npm\node_modules\gatsby\bin\gatsby-serve.js to cli-serve.js
C:\Users\aniru\AppData\Roaming\npm\node_modules\gatsby\bin\gatsby-build.js to cli-build.js 
C:\Users\aniru\AppData\Roaming\npm\node_modules\gatsby\bin\gatsby-new.js to cli-new.js

**Comment by awesomeaniruddh** [comment]
@MichaelDemchenko That fixed it! Thanks a lot. Why don't you open a PR to add this to the readme until this gets fixed?

**Comment by suenot** [comment]
:+1: have the save problem in windows 7

**Comment by KyleAMathews** [comment]
[This PR in commander.js](https://github.com/tj/commander.js/pull/327) seems related. There's been other people reporting problems with git-style subcommands on Windows.

What version of node are y'all using? Curious if it's only older versions causing trouble.

**Comment by MichaelDemchenko** [comment]
I use v4.2.2

**Comment by mudrummer012** [comment]
Node v4.2.3 on Windows 10

I'm getting this as of Gatsby 0.7.7 so this is still an issue.

**Comment by SaintPeter** [comment]
Same issue on Windows 10, Node v4.2.2, Gatsby 0.7.7

Renaming the files [as described above](https://github.com/gatsbyjs/gatsby/issues/80#issuecomment-155157743) fixed the issue for me.

Thanks @MichaelDemchenko

**Comment by cable729** [comment]
Same issue on Windows 10, Node v5.5.0, Gatsby 0.7.7. Although, I had to update different files:

```
%appdata%\npm\node_modules\gatsby\bin\cli.js => cli-serve.js
%appdata%\npm\node_modules\gatsby\bin\gatsby-build.js => cli-build.js 
%appdata%\npm\node_modules\gatsby\bin\gatsby-new.js => cli-new.js
```

**Comment by KyleAMathews** [comment]
Hey peeps, @NogsMPLS submitted [a PR which *should* fix this issue](#159). I released 0.8.5 with his PR included — could some of you install this and report back if Gatsby does indeed now work correctly on Windows?

**Comment by KyleAMathews** [comment]
Closing this out. Please comment if this problem still exists or open a new issue if you discover other problems on Windows.
